models/device_os.py

```diff
@@ -0,0 +1,66 @@
+from sets import Set
+import json
+
+from ..base_request import BaseRequest
+from ..resources import Message
+from ..settings import Settings
+import resin.exceptions as ResinException
+
+NETWORK_WIFI = 'wifi'
+NETWORK_ETHERNET = 'ethernet'
+
+NETWORK_TYPES = [
+	NETWORK_WIFI,
+	NETWORK_ETHERNET
+]
+
+VALID_OPTIONS = [
```

Comment:
This module should ideally start using `device-type.json`. It can be fetched from the `API/config` endpoint